src/main/java/net/pms/parsers/JaudiotaggerParser.java

```diff
@@ -199,7 +200,7 @@ public static void parse(File file, MediaAudioMetadata audioMetadata) {
 			}
 			addMusicBrainzIDs(af, audioMetadata);
 			addAudioTrackRating(af, audioMetadata);
-		} catch (IOException | CannotReadException | InvalidAudioFrameException | ReadOnlyFileException | TagException e) {
+		} catch (IOException | CannotReadException | InvalidAudioFrameException | NullBoxIdException | ReadOnlyFileException | TagException e) {
```

Comment:
This is why I like the old exception syntax of `Exception e`, and I often refactor this multicatch syntax to the old one - for example here recently https://github.com/UniversalMediaServer/UniversalMediaServer/commit/db76342bc677350f691a91eb5895fdde42feb002#diff-a290dd06f8ac9f7150c0d0cf38c4debd445772b457a5814b060f757c6ee9c474R180-R183. It's just nicer to know you will catch all exceptions IMO. What do you think?

Reply:
Thing is that custom exception implementation (eg `org.jaudiotagger.audio.exceptions.CannotReadException`) should **ideally** extends an internal Exception (eg `org.jaudiotagger.JaudioTaggerException`) then `java.lang.Exception` instead of directly extends `java.lang.Exception`

Then, you catch everything with `org.jaudiotagger.JaudioTaggerException`.

I changed code.